README.md

```diff
@@ -1,2 +1,57 @@
-# split-expense
+# About
+
 A website to split expenses based on invoice
+
+# Local setup
+
+Application is built using python & flask-restplus as the backend and react for the front end. following are the steps required to setup the application in local computer
+
+## Backend setup
+
+Backend application uses postgres as the dependency, follow the steps to install the postgres through docker
```

Comment:
- It's will be nice if we add links to the **technical terms**. 
- Each bullet point statement should start with the **Uppercase letter**.

Reply:
Thanks for the review, will update the requested changes :)